packages/dendron-next-server/components/graph-filter-view.tsx

```diff
@@ -112,6 +122,59 @@ const GraphFilterView = ({ config, setConfig }: FilterProps) => {
   );
 };
 
-const FilterViewInput = () => {};
+const FilterViewStringInput = ({
+  fieldKey,
+  label,
+  entry,
+  updateConfigField,
+  nodeCount,
+}: {
+  fieldKey: string;
+  label: string;
+  entry: GraphConfigItem<string>;
+  updateConfigField: (key: string, value: string | number | boolean) => void;
+  nodeCount: number;
+}) => {
+  const [updateTimeout, setUpdateTimeout] =
+    useState<NodeJS.Timeout | null>(null);
+
+  // This timeout is designed to maximize filter responsiveness while minimizing hang times
+  const handleChange = (newValue: string) => {
+    const delay = nodeCount < 100 ? 0 : 400;
+
+    if (updateTimeout) clearTimeout(updateTimeout);
+
+    setUpdateTimeout(
+      setTimeout(() => {
+        updateConfigField(fieldKey, newValue);
+
+        setUpdateTimeout(null);
+      }, delay)
+    );
+  };
+
+  return (
+    <Space direction="vertical" style={{ marginBottom: "1rem" }}>
+      <Typography>{label}</Typography>
+      <Input
+        defaultValue={entry.value}
+        onChange={(newValue) => handleChange(newValue.target.value)}
```

Comment:
Looks like c/p isn't working in the webview because of the workaround we have: [[Copy and Paste|pkg.dendron-next-server.arch#copy-and-paste]]
Can we make sure that keyboard events are propagated upwards to vscode? You can see an working example of this is `Dendron: Configure` in the webview there.

Reply:
The keyboard events are now being propagated correctly, but despite that they still aren't triggering commands like Copy and Paste. Because of that this is a bit blocked at the moment - every other component of the PR should be good to go